spyder/dependencies.py

```diff
@@ -113,3 +135,66 @@ def missing_dependencies():
         return status(deps=missing_deps, linesep='<br>')
     else:
         return ""
+
+
+def declare_dependencies():
+    add("cloudpickle", "cloudpickle",
+        _("Serialize variables in the IPython kernel to send to Spyder."),
+        required_version=CLOUDPICKLE_REQVER)

```

Comment:
Instead of calling `add` for each dependency, I think we should create a list of dicts out of this function called `DEPENDENCIES` (placed below the REQVER constants) with the following contents:

```python
DEPENDENCIES = [
    {'modname': "cloudpickle",
     'package_name': "cloudpickle",
     'features': _("Serialize variables in the IPython kernel to send to Spyder."),
     'required_version': CLOUDPICKLE_REQVER},
    {'modname': "pygments",
     'package_name': "pygments",
     'features': _("Syntax highlighting for Matlab, Julia and other file types"),
     'required_version': PYGMENTS_REQVER},
    ...
]
```

Then here in `declare_dependencies` we could simply do the following:

```python
def declare_dependencies():
    for dep in DEPENDENCIES:
        add(dep['modname'], dep['package_name'],
            dep['features'], dep['required_version'])
```